oci-library/oci-hol/java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -23,21 +23,44 @@ In this lab, you will:
 * You are using an Oracle Linux image on your host machine or compute instance for this workshop.
 * Access to the cloud environment and resources configured in Lab 2
 
-## Task 1: Transfer of files to Compute Instance
+## Task 1: Prepare agent software and response file for Management Agent installation
 
-1. The last line in the response file `Service.plugin.jms.download=true` will download and enable the JMS plugin
+1. Connect to your instance using SSH.
 
-  ![image of input rsp file](/../images/input-rsp-updated.png)
+2. Enter the following command to download the management agent software file via wget into the remote host compute instance.
 
-2. Open up a **Terminal** or **Command Prompt** window in the local machine where the install key and management agent software file is saved
+  ### For **Linux**
+  Download the management agent software using wget:
 
-3. Enter the following command to transfer the install key and management agent software file via scp into the remote host compute instance
     ```
     <copy>
-    scp <full_path_of_file_to_be_transfered_on_local_host> opc@<public_IP_Address>:<full_path_of_remote_directory_transferred_to>
+    wget https://objectstorage.us-ashburn-1.oraclecloud.com/n/idtskf8cjzhp/b/installer/o/Linux-x86_64/latest/oracle.mgmt_agent.rpm
     </copy>
     ```
 
+  ### For **other operating systems**
+  Replace the link with the link for your operating system found on the [official documentation](https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html). Under **Download the Agent Software Using a URL**, **right-click** and click **Copy Link Address** to copy the correct link.
+
+  ![image of obtaining link for management agent software](/../images/link-management-agent-software.png)
+
```

Comment:
It could be a good place to explain the response file and install key here because there is mention of both and might confuse the user. Task 1 point 3